src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -446,7 +446,7 @@ void bmi160SpiGyroInit(gyroDev_t *gyro)
     bmi160IntExtiInit(gyro);
 #endif
 
-    spiSetClkDivisor(dev, spiCalculateDivider(BMI160_MAX_SPI_CLK_HZ));
```

Comment:
Does this work before?

Reply:
I checked master on GitHub and I am seeing different code.
<img width="1373" alt="Snímek obrazovky 2022-06-29 v 23 50 38" src="https://user-images.githubusercontent.com/36531759/176551526-50630dd4-c937-4024-9d33-38dfdaac0279.png">

Reply:
@SteveCEvans did you rebase before starting as this line should be on 440https://github.com/betaflight/betaflight/blob/81a3c53b1cc85e91d5cd4be2544d3ca46eca5237/src/main/drivers/accgyro/accgyro_spi_bmi160.c#L440

Reply:
Indeed a rebase was required, but a merge must have screwed up the BMI160 driver. Now sorted. This is the issue with having code which doesn’t get built. It can get broken and nobody spots it.